mis/shared/types.py

```diff
@@ -108,21 +111,23 @@ class MISSolution:
     nodes: list[int]
     frequency: float
     weight: float = field(init=False)
+    size: int = field(init=False)
 
     def __post_init__(self) -> None:
         # Consistency check: nodes from the list must be distinct.
         assert len(self.nodes) == len(set(self.nodes)), "All the nodes in %s should be distinct" % (
             self.nodes,
         )
         self.weight = calculate_weight(self.original, self.nodes)
+        self.size = len(self.nodes)
 
-    def draw(self, node_size: int = 600, highlight_color: str = "red") -> None:
+    def draw(self, node_size: int = 600, highlight_color: str = "darkgreen") -> None:
```

Comment:
Weren't we thinking about changing the font, too?

Reply:
Yep, I deliberately left it out. Will do now. Another parameter here is fine ?

Reply:
Done.